f5/bigip/tm/cm/trust.py

```diff
@@ -54,8 +58,12 @@ class Remove_From_Trust(UnnamedResourceMixin, CommandExecutionMixin, Resource):
 
     def __init__(self, cm):
         super(Remove_From_Trust, self).__init__(cm)
+        base_uri = type(self).__name__.replace('_', '-').lower()
```

Comment:
Regrettably this will not work due to custom endpoint. I had this using get uri from pathelement and it was failing. Try it :)